core/templates/pages/community-dashboard-page/translation-opportunities/translation-opportunities.directive.ts

```diff
@@ -144,18 +147,24 @@ angular.module('oppia').directive(
                       $scope.moreAvailable = textAndAvailability.more;
                       $scope.loadingData = false;
                     });
+
                   $scope.skipActiveTranslation = function() {
-                    $scope.textToTranslate = (
-                      TranslateTextService.getTextToTranslate().text);
+                    var textAndAvailability = (
+                      TranslateTextService.getTextToTranslate());
+                    $scope.textToTranslate = textAndAvailability.text;
+                    $scope.moreAvailable = textAndAvailability.more;
                     $scope.activeWrittenTranslation.html = '';
                   };
+
                   $scope.suggestTranslatedText = function() {
                     if (!$scope.uploadingTranslation && !$scope.loadingData) {
                       $scope.uploadingTranslation = true;
                       TranslateTextService.suggestTranslatedText(
                         $scope.activeWrittenTranslation.html,
                         TranslationLanguageService.getActiveLanguageCode(),
                         function() {
+                          AlertsService.addSuccessMessage(
+                            'Submitted suggestion!');
```

Comment:
I think this should say "Submitted translation for review" as the person submitting the translation won't be aware of what a suggestion is.

Reply:
(Also, the toast for question opportunities should also be modified to remove the term suggestion in my opinion)

Reply:
Done!